modules/nodes-pods-vertical-autoscaler-tuning.adoc

```diff
@@ -20,7 +20,165 @@ You can perform the following tunings on the VPA components by editing the `Vert
 
 * To configure the VPA Operator to monitor only workloads that are being managed by a VPA CR, set the `memory-saver` parameter to `true` for the recommender component.
 
-The following example VPA controller CR sets the VPA API QPS and burts rates, configures the component pod resource requests, and sets `memory-saver` to `true` for the recommender:
+For guidelines on the resources and rate limits that you could set for each VPA component, the following tables provide baseline values, depending on the size of your cluster and other factors. 
+
+[IMPORTANT]
+====
+These values were derived by internal Red{nbsp}Hat testing on clusters that are not necessarily representative of the loads a real-world cluster would experience. You should test these values in a non-production cluster before configuring a production cluster. 
+====
+
+.Requests by containers in the cluster
+[cols="1,1,1,1,1,1,1,1,1,3,3"]
+|===
+| Component 2+| 1-500 containers 2+| 500-1000 containers 2+| 1000-2000 containers 2+| 2000-4000 containers 2+| 4000+ containers
+
+| 
+| *CPU*
+| *Memory*
+| *CPU*
+| *Memory*
+| *CPU*
+| *Memory*
+| *CPU*
+| *Memory*
+| *CPU*
+| *Memory*
+
+s| Admission 
+| 25m 
+| 50Mi
+| 25m 
+| 75Mi 
+| 40m 
+| 150Mi 
+| 75m 
+| 260Mi 
+| (0.03 x c)/2 + 10 ^[1]^
+| (0.1 x c)/2 + 50 ^[1]^
+
+s| Recommender
+| 25m
+| 100Mi
+| 50m
+| 160Mi
+| 75m
+| 275Mi
+| 120m
+| 420Mi
+| (0.05 x c)/2 + 50 ^[1]^
+| (0.15 x c)/2 + 120 ^[1]^
+
+s| Updater
+| 25m
+| 100Mi
+| 50m
+| 220Mi
+| 80m
+| 350Mi
+| 150m
+| 500Mi
+| (0.07 x c)/2 + 20 ^[1]^
+| (0.15 x c)/2 + 200 ^[1]^
+
+|===
+[.small]
+. `c` is the number of containers in the cluster.
+
+[NOTE]
+====
+It is recommended that you set the memory limit on your containers to double the recommended requests in the table. However, because CPU is a compressible resource, setting CPU limits for containers can throttle the VPA. As such, it is recommended that you do not set a CPU limit on your containers.
+====
+
+.Rate limits by VPAs in the cluster
+[cols="1,2,2,2,1,2,1,2,1"]
```

Comment:
I'm not sure if you can change this, but the columns are not the same size in the preview:
![image](https://github.com/user-attachments/assets/9336ce04-aac8-4be5-ad7e-299d4edfa9a0)

If it's automatically sized then don't worry about it.

Reply:
@maxcao13  This any better? If I make all the columns the same, the footnote thingy [1] rolls to the next line in my preview. Here, all the QPS and all the Burst columns are the same.

![image](https://github.com/user-attachments/assets/16331242-e787-4a45-84c4-c436473161a8)

Reply:
Looks good :-)